packages/hardhat/contracts/FundingVault.sol

```diff
@@ -49,18 +49,18 @@ contract FundingVault{
 
     // State Variables //
     using SafeERC20 for IERC20;
-    IERC20 public immutable participationToken;
-    uint256 public participationTokenAmount;
-    uint256 public timestamp;
-    uint256 public immutable minFundingAmount; //The minimum amount of ETH required in the contract to enable withdrawal.
-    uint256 public exchangeRate; //The exchange rate of ETH per token
-    address public withdrawalAddress; //WithdrawalAddress is also considered as owner of the Vault. 
-    address private developerFeeAddress; //develper address
-    uint256 private developerFeePercentage; 
-    string  public projectURL;
-    string public projectTitle;
-    string public projectDescription;
-    bool private minFundingReached;
+    IERC20 public immutable participationToken; // The token that will be used as participation token to incentivise contributions
+    uint256 public participationTokenAmount; // The initial  participation token amount which will be in fundingVault
+    uint256 public timestamp; // The date limit until which withdrawal or after which refund is allowed.
+    uint256 public immutable minFundingAmount; // The minimum amount of ETH required in the contract to enable withdrawal.
+    uint256 public exchangeRate; // The exchange rate of ETH per token
+    address public withdrawalAddress; // WithdrawalAddress is also considered as owner of the Vault. 
+    address private developerFeeAddress; // Developer address
+    uint256 private developerFeePercentage; // Developer percentage in funds collected
+    string  public projectURL; // A link or hash containing the project's information (e.g., GitHub repository).
+    string public projectTitle; // Name of the Project
+    string public projectDescription; // Short description of the project
+    bool private minFundingReached; // Variable to know if minimum funding raised or not
```

Comment:
Perhaps it would be better to replace `minFundingReached` by `uint256 amountRaised`. Then it would be easier for us to know how much has been raised.